cirq-core/cirq/contrib/__init__.py

```diff
@@ -24,3 +24,4 @@
 from cirq.contrib.qcircuit import circuit_to_latex_using_qcircuit as circuit_to_latex_using_qcircuit
 from cirq.contrib import json  # noqa: F401
 from cirq.contrib.circuitdag import CircuitDag as CircuitDag, Unique as Unique
+from cirq.contrib import noise_models as noise_models
```

Comment:
We should try to avoid it here.  If cirq.contrib registers its class resolver here, the import of noise_models can be deferred to the resolver execution.

Reply:
done